drivers/kscan/Kconfig.matrix

```diff
@@ -0,0 +1,83 @@
+# Keyboard scan matrix configuration options
+
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config KSCAN_MATRIX
+	bool "keyboard scan matrix"
+	help
+	  A matrix circuit is used in most computer keyboards where the key
+	  switches are connected by a grid of wires, similar to a diode matrix.
+
+
+if KSCAN_MATRIX
+
+config KSCAN_MATRIX_MAX_COLUMNS
+	int "keyboard matrix max column"
+	default 18
+	help
+	  Maximum columns of the matrix circuit.
+
+config KSCAN_MATRIX_MAX_ROWS
+	int "keyboard matrix max row"
+	default 8
+	help
+	  Maximum rows of the matrix circuit.
+
+config KSCAN_MATRIX_DRIVER
+	bool "Keyboard matrix driver for scanning"
+	help
+	  Keyboard Scan Matrix Driver Support
+
+if KSCAN_MATRIX_DRIVER
+
+config KSCAN_MATRIX_DRV_NAME
```

Comment:
Please add some help text for this option.

Reply:
Done.